lib/ReactTags.js

```diff
@@ -56,6 +56,42 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.onPaste) {
+      return this.props.onPaste(e)
+    }
+
+    const { delimiterChars } = this.props
+
+    e.preventDefault()
+
+    // get the text data from the clipboard
+    const data = e.clipboardData.getData('Text')
+
+    if (data && delimiterChars.length > 0) {
+      // split the string based on the delimiterChars as a regex
+      const tags = data.split(new RegExp(delimiterChars.join('|'), 'g'))
```

Comment:
Doing some tests, the original `data.split(new RegExp(delimiterChars.join('|'), 'g'))` seems more robust than `data.split(new RegExp('[' + delimiterChars.join('') + ']'))`.

Results:

```
delimiterChars = ['^',',',';']
data = 'green; blue, white'
data.split(new RegExp(delimiterChars.join('|'), 'g'))
// result: ["green", " blue", " white"]
data.split(new RegExp('[' + delimiterChars.join('') + ']'))
// result: ["", "", "", "", "", ";", "", "", "", "", ",", "", "", "", "", "", ""]
```
I am tempted to leave the original code based on this test.